test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -848,6 +848,11 @@ UCS_TEST_P(test_ucp_mmap, fixed) {
     ucs_status_t status;
     bool         is_dummy;
 
+    if (has_transport("cuda_ipc") ||
+        has_transport("cuda_copy")) {
+        UCS_TEST_SKIP_R("UCP_MEM_MAP_FIXED test is unsupported with CUDA transport");
+    }
+
```

Comment:
pls remove this, it should be fixed by different fixed address

Reply:
this worksaround another issue not related to BAR1 size

Reply:
Do you want this as a separate patch ?

Reply:
yes